README.rst

```diff
@@ -91,14 +91,12 @@ In submission mode you can view the self text for a submission and browse commen
 Configuration
 -------------
 
-RTV will read a configuration file located at **~/.rtv**.
+RTV will read a configuration file located at ``$XDG_CONFIG_HOME/rtv/rtv.cfg``.
```

Comment:
Change to ``~/.config/rtv/rtv.cfg``? I'm worried that a lot of people won't know what $XDG_CONFIG_HOME stands for, or that it defaults to ``~/.config`` if not set.